templates/candlepin.service.erb

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Candlepin
+Wants=syslog.service
+
+[Service]
+Restart=always
+RestartSec=30
+TimeoutStartSec=0
+TimeoutSec=300
+ExecStartPre=-/usr/bin/podman rm "candlepin-1"
+ExecStart=/usr/bin/podman run --name candlepin-1 --env-file /etc/sysconfig/candlepin --net host -v /etc/candlepin:/etc/candlepin:z -v /etc/tomcat:/etc/tomcat:z -p <%= scope['candlepin::ssl_port'] %> <%= scope['candlepin::container_image'] %>:<%= scope['candlepin::container_version'] %>
+ExecReload=-/usr/bin/podman stop "candlepin-1"
+ExecReload=-/usr/bin/podman rm "candlepin-1"
```

Comment:
Shouldn't reload be used for "make the service reload its configuration without stopping it" kind of thing?

Reply:
I think if you make any configuration changes on disk, or environment variables you have to stop the container and re-start to pick up the new. I could be wrong though :)

Reply:
Sounds more like restart and reload should simply not be implemented then.